openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -256,7 +256,7 @@ def _run_MD(simulation: openmm.app.Simulation,
                 temperature: unit.Quantity,
                 barostat_frequency: unit.Quantity,
                 timestep: unit.Quantity,
-                equil_steps_nvt: int,
+                equil_steps_nvt: Optional[int],
```

Comment:
also update the docstring below to reflect this change

Reply:
Updated the doc strong (and also in the openmm settings)